src/js/actions/export.js

```diff
@@ -74,41 +71,82 @@ define(function (require, exports) {
      *
      * @private
      * @param {number} documentID
-     * @param {number} layerID
+     * @param {number=} layerID optional, if supplied, sync doc-level metadata
      * @return {Promise}
      */
-    var _syncLayerExportMetadata = function (documentID, layerID) {
+    var _syncExportMetadata = function (documentID, layerID) {
         var documentExports = this.flux.stores.export.getDocumentExports(documentID),
-            layerExports = documentExports && documentExports.getLayerExports(layerID),
-            layerExportsArray = layerExports && layerExports.toJS(),
-            document = this.flux.stores.document.getDocument(documentID),
-            layer = document && document.layers.byID(layerID),
-            exportEnabled = document && layer && layer.exportEnabled;
-
-        var exportsMetadata = {
-            exportAssets: layerExportsArray,
-            exportEnabled: exportEnabled === undefined ? false : exportEnabled
-        };
+            exportsMetadata,
+            playObject;
+
+        if (layerID || layerID === 0) {
```

Comment:
I wasn't sure if layerIDs could legitimately be zero, but if so, then a simple `if layerID` test will return false.  because javascript.